web/styles/app_components.css

```diff
@@ -1221,14 +1222,18 @@ div.overlay {
         align-items: center;
         box-sizing: border-box;
         width: 100%;
-        height: 100%;
-        margin: 1px;
+        height: 90%;
+        margin: 1.5px;
         cursor: pointer;
         border-radius: 5px;
 
         &:hover:not(.selected-tab) {
-            background-color: hsl(0deg 0% 100%);
+            background-color: hsl(0deg 0% 100% / 60%);
```

Comment:
This PR presents a good opportunity to set CSS variables for the color values around this area, and fits with the "improve the CSS" theme of the PR, too.

Reply:
Added a prep commit